main.js

```diff
@@ -173,6 +178,8 @@ var start_local_server = function() {
       serverRunning = false;
     });
     subpy.unref();
+  } else {
+    mainWindow.webContents.executeJavaScript("console.log('Unable to find OpenBazaar-Server at: '" + serverPath + "')");
```

Comment:
es-lint is working and is flagging about 20 other files. I'm not fixing those in this PR as it was tiny. I will resolve the main.js ones of which there are still a lot I didn't commit, but whatever.